src/Datadog.Trace.Core/Datadog.Trace.Core.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- NuGet -->
+    <Version>0.1.0-prerelease</Version>
+    <Title>Context Storage for Datadog APM</Title>
+    <Description>Context storage and management for Datadog .NET Tracer library.</Description>
+  </PropertyGroup>
+
+</Project>
```

Comment:
We may want to add `<RootNamespace>Datadog.Trace</RootNamespace>` so IDEs don't keep suggesting we change all the namespaces in this project to `Datadog.Trace.Core`.